server/utils/validation/signUpSchema.ts

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+
+const signUpSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().max(250).min(4).email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } })
+    .required(),
+  password: Joi.string().max(50).min(3).required(),
+  address: Joi.string().required(),
+  phone: Joi.string().required(),
+  is_admin: Joi.boolean(),
```

Comment:
There is no need to validate is_admin because here you just want to validate Sign Up form inputs.